### ISSUE TITLE
Julia throwing weird error

### ISSUE THREAD

**Issue by TheFGFSEagle** [abassign/Photoscenary#40]
I have just installed Julia, then `git clone`d this repository, then run `julia photoscenary.jl --help`. Output is following:
```sh
fatal: error thrown and no exception handler available.
#<null>
jl_errorf at /home/user/.local/lib/x86_64-linux-gnu/libjulia.so.1 (unknown line)
jl_load_dynamic_library at /home/user/.local/lib/x86_64-linux-gnu/libjulia.so.1 (unknown line)
unknown function (ip: 0x7fd6aa7aca02)
jl_load_and_lookup at /home/user/.local/lib/x86_64-linux-gnu/libjulia.so.1 (unknown line)
jlplt_pcre2_compile_8_2341 at /home/user/.local/lib/x86_64-linux-gnu/julia/sys.so (unknown line)
compile at ./pcre.jl:120
compile at ./regex.jl:72
#occursin#363 at ./regex.jl:172 [inlined]
occursin at ./regex.jl:172 [inlined]
isdirpath at ./path.jl:117 [inlined]
normpath at ./path.jl:337
abspath at ./path.jl:383
abspath at ./path.jl:391
__init__ at ./sysinfo.jl:119
jfptr___init___14051 at /home/user/.local/lib/x86_64-linux-gnu/julia/sys.so (unknown line)
unknown function (ip: 0x7fd6aa8544d1)
unknown function (ip: 0x7fd6aa83f17b)
unknown function (ip: 0x55f4e19f559a)
__libc_start_main at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
unknown function (ip: 0x55f4e19f564d)
```
`julia --version` gives
```sh
julia version 1.4.1
```

**Comment by abassign** [comment]
Hello,
Versioned i Julia (1.4.1) is too old the program is tested for version 1.6.x but also from version 1.5.4 other users said it works. Here is the Wiki article explaining the problem: https://wiki.flightgear.org/Julia_photoscenery_generator#Julia_installation
Therefore the only possibility is to update Julia, which is sometimes complicated because the repositories are not always very up to date.
For this follow this procedure:
https://medium.com/coffee-in-a-klein-bottle/install-julia-1-5-on-ubuntu-bb8be4b2571d
While it doesn't appear from the guide, this solution also works on Debian.
For other Linux distributions the binary you extracted from here: https://julialang.org/downloads/ should always work, the only caveat is to make an appropriate soft link.
Let me know what version of Linux you are using and if the answer solved the problem.

Ciao, Adriano

**Comment by TheFGFSEagle** [comment]
Yes, this fixed it. I am on Ubuntu Studio 20.04. But one question - there is another photoscenery generator available written in Python. When I try to download photoscenery from ArcGIS using that script, I get a 500 Server Error. I went through your code and observed that your script is using the same URL for downloads from ArcGIS. Does your script work with the ArcGIS provider right now ?

**Comment by abassign** [comment]
In params.xml there are configurations you can use, and add others that may interest you. I would like to have other servers to propose for all of us who use this program.
Error 500 indicates that the server was not reached so there can be two reasons:
The server is actually unreachable (I use 1 and 2, does not know Spanish 3 and Polish 4 if they are always operational) or that server is blocked for your country, often happens with US servers if the country you are calling from does not it pleases them.
Where are you calling from?
However, if you want to try, turn on the debug option --debug 2
At this point the actual calls made to the servers will be shown. You can check in this way, by entering them in a browser, if you download the image.

**Comment by TheFGFSEagle** [comment]
No no, you got me completely wrong. The thing that wonders me is that for exactly the same region (for example tile 3122034), your script works perfectly fine, but another Python script that also downloads photoscenery (but is considerably faster, when it works ??) does _not_ work when trying to download too small images (they are merged to one big tile in the end). But you need not bother with this - it's the author of the Python script who needs to bother with this !
